shipshape.yml

```diff
@@ -86,9 +86,9 @@ checks:
         - key: error_level
           value: hide
     - name: '[FILE] Detect module files in theme folder'
-      pattern: '.*.info.yml'
+      pattern: '.*\.info\.yml$'
       ignore-missing: true
-      path: 'themes'
+      path: web/themes
```

Comment:
So this one will work locally and on PaaS I guess, but for SaaS this path needs to be `themes` due to the dir structure.